applications/sofa/gui/qt/QDisplayPropertyWidget.cpp

```diff
@@ -161,7 +161,7 @@ void QDisplayPropertyWidget::addComponent(const QString& component, core::object
             continue;
 
         // for each data of the current object we determine where it belongs
-        QString group = data->getGroup();
+        QString group = data->getGroup().c_str();
```

Comment:
normally there is a method from Qt to convert directly std::string to QString. Maybe better than recasting the string to const char* before giving it ton QString.
(probably it does the same internally, but it is less dirty)

Reply:
Makes sense, I'll check for that Qt method